specification_templates/vaccine.yaml

```diff
@@ -1,19 +1,19 @@
 data:
-  regression_version: '2020_09_15.01'
-  covariate_version: '2020_09_10.03'
+  regression_version: '2020_11_15.02'
+  covariate_version: '2020_11_15.02'
```

Comment:
I'm obviously not an expert on this codebase. Are all of these yaml files hand-entered?

Reply:
Yes.  In lieu of having about 30 cli args, the options are all specified here.